src/backend/src/services/teams.services.ts

```diff
@@ -418,6 +427,47 @@ export default class TeamsService {
     return teamTypes;
   }
 
+  /**
+   * Changes the description of the given teamType to be the new description
+   * @param user The user who is editing the description
+   * @param teamTypeId The id for the teamType that is being edited
+   * @param name the new name for the team
+   * @param iconName the new icon name for the team
+   * @param description the new description for the team
+   * @param organizationId The organization the user is currently in
+   * @returns The team with the new description
+   */
+  static async editTeamType(
+    user: User,
+    teamTypeId: string,
+    name: string,
+    iconName: string,
+    description: string,
+    filePath: string | null,
+    organizationId: string
+  ): Promise<TeamType> {
+    if (!isUnderWordCount(description, 300)) throw new HttpException(400, 'Description must be less than 300 words');
+
+    if (!(await userHasPermission(user.userId, organizationId, isAdmin)))
+      throw new AccessDeniedException('you must be an admin to edit the team types description');
+
+    const currentTeamType = await prisma.team_Type.findUnique({
+      where: { teamTypeId }
+    });
+
```

Comment:
Throw an exception if the team type to edit doesnt exist, this will also let you get rid of the ? operator on 464